content/doc/feedback-form.adoc

```diff
@@ -0,0 +1,143 @@
+---
+layout: documentation
+title: Jenkins User Documentation
+section: doc
+---
+
+////
+This feedback form was prepared from a Google Form, based on the techniques in
+the following documentation sources:
+https://codepen.io/learningcode/post/customize-a-google-form-for-your-website,
+http://mrhaki.blogspot.com.au/2014/06/awesome-asciidoc-include-raw-html.html,
+http://www.freshtechtips.com/2013/05/custom-google-drive-contact-form.html,
+https://stackoverflow.com/questions/24436165/set-value-of-hidden-input-inside-form
+////
+
+++++
+<!-- On loading up the window's content, this JavaScript code stores the value
+     of the previous page's (i.e. referrer) URL in the variable
+     "feedbackPageUrl" in the browser's local storage. -->
+<script>
```

Comment:
Lots of this code looks redundant with `_feedback-form.adoc`.

I'm going to approve this code for now assuming it's temporary. If we decide to keep this code after a month, then I would expect @gilesgas to refactor the duplication into proper `.js` files.